packages/experimental/RadioGroup/src/Radio/RadioTokens.ts

```diff
@@ -0,0 +1,67 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+
+import { RadioTokens } from './Radio.types';
+
+export const defaultRadioTokens: TokenSettings<RadioTokens, Theme> = (t: Theme) =>
```

Comment:
Just a general note that while I recognize we don't have redlines, generally Fluent 2 controls have rounded corners, so that might be one exception to the "stick with the old visuals" rule

Reply:
Would you suggest adding rounded corners to the focusable border that appears on the selected Radio?

Reply:
Yeah, that's what I would expect to see.

Reply:
Replaced with enableFocusRing and added rounded corners.